ci-operator/step-registry/openshift/manifests/external/openshift-manifests-external-commands.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+LOCALPATH="${SHARED_DIR}/manifest_external.yaml"
```

Comment:
`manifests_external.yaml` to better match the step name

Reply:
Installer step requires all files to start with `manifest_` (no `s`) to be accepted

Reply:
>I'm assuming that URL may contain multiple manifests/YAML docs in a single file

Right, that's what I'll experiment on next - UUIC multidoc yaml won't be accepted, so this step could split it into multiple files